SwiftyJSON.xcodeproj/project.pbxproj

```diff
@@ -672,7 +684,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
```

Comment:
I believe it because for iOS7, we have to include the xcode project or drag and drop in the .swift files for it to work.
where as with cocoapods, they can only do `use_frameworks!` with swift libs.
So it's fine to set it to iOS8+.